src/stylesheets/components/_avatar_image.scss

```diff
@@ -37,6 +37,30 @@
   }
 }
 
+.avatar-image-rectangle {
+  width: 160px;
+  height: 100px;
+  border-radius: 5px;
+
+  &-wrapper {
+    border-radius: 5px !important;
+    width: 160px;
+    height: 100px;
+    flex-shrink: 0;
+    display: grid;
+    place-items: center;
+    position: relative;
+  }
+
+  &--selected {
+    border: 4px solid #8ecb58;
```

Comment:
can we use a variable for this color

Reply:
I actually pasted this from the css above it in the same file it should be updated there as well, I wanted these styles to be polished by pratheesh
I think he would like to adjust sizing and spacing of the upload buttons and error message spans

Reply:
--selected-border-color added